README.md

```diff
@@ -4,18 +4,119 @@ Go-Attestation
 [![GoDoc](https://godoc.org/github.com/google/go-attestation/attest?status.svg)](https://godoc.org/github.com/google/go-attestation/attest)
 
 Go-Attestation abstracts remote attestation operations across a variety of platforms
-and TPMs.
+and TPMs, enabling remote validation of machine identity and state. This project
+attempts to provide high level primatives for both client and server logic.
 
+Talks on this project:
+
+* _"Making Remote Attestation Useful on Linux"_ - Linux Security Summit - September 2019 - ([Video](https://www.youtube.com/watch?v=TKva_h66Ptc) 26min)
+* _"Making Device Identity Trustworthy"_ - Open Source Summit Europe - October 2019 - ([Slides](https://static.sched.com/hosted_files/osseu19/ec/Device%20Identity.pdf))
+
+## Status
+
+Go-Attestation is under active development and **is not** ready for production use. Expect
+API changes at any time.
+
+Please note that this is not an official Google product.
 
 ## Installation
 
 The go-attestation package is installable using go get: `go get github.com/google/go-attestation/attest`
 
 Linux users must install `libtspi` and its headers. This can be installed on debian-based systems using: `sudo apt-get install libtspi-dev`.
 
-## Status
+## Example: device identity
 
-Go-Attestation is under active development and **is not** ready for production use. Expect
-API changes at any time.
+TPMs are provisioned with a set of Endorcement Keys (EK) by the manufacturer.
+These optionally include a certificate signed by the manufacturer and act as a
+TPMs identity. For privacy reasons the EK can't be used to sign or encrypt data
```

Comment:
The EK usually cant be used, but the spec now permits it & we expect some cloud vendors to provide EKs which can sign in the near future.

Reply:
done